kpi/deployment_backends/kc_access/utils.py

```diff
@@ -30,7 +30,7 @@ def _trigger_kc_profile_creation(user):
     UserProfile if none exists already
     """
     url = settings.KOBOCAT_URL + '/api/v1/user'
-    token, _ = Token.objects.get_or_create(user=user)
+    token, _ = Token.objects.using('kobocat').get(user=user)
```

Comment:
I thought `get_or_create` was a function that returns a tuple which you can destructure to have `token` and `_`, no? I'd like to think `get` returns only one value which is the Token.

Reply:
Ah...yes, let me change that